### PR TITLE
Add imagepruners resource

### PR DESCRIPTION
**Pull request by wzheng1**
Add imagepruners resource.

### CODE REVIEW

File: lib/openshift/imagepruners_imageregistry_operator_openshift_io.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'openshift/cluster_resource'
+
+module BushSlicer
+  class ImageprunersImageregistryOperatorOpenshiftIo < ClusterResource
```
Comment (pruan-rht) [inline lib/openshift/imagepruners_imageregistry_operator_openshift_io.rb:4]: Do we have to use such a long class name?

Comment (wzheng1) [reply lib/openshift/imagepruners_imageregistry_operator_openshift_io.rb:4]: I just copy from other resources, is it fine to define shorter?

Comment (wzheng1) [reply lib/openshift/imagepruners_imageregistry_operator_openshift_io.rb:4]: In order to be consistent with other resources, I suggest to keep it like this.

Comment (pruan-rht) [reply lib/openshift/imagepruners_imageregistry_operator_openshift_io.rb:4]: @wzheng1 that's a lot of typing... I think it's fine for the `RESOURCE=...` line but for the class name, I think `ImagePruners` is sufficient.  WDYT @akostadinov ?

Comment (akostadinov) [reply lib/openshift/imagepruners_imageregistry_operator_openshift_io.rb:4]: I would even call it `ImagePrunersImageregistryOperatorOpenshiftIo`, then the short name will be `image_prunner` which reads much better.

Comment (akostadinov) [reply lib/openshift/imagepruners_imageregistry_operator_openshift_io.rb:4]: Using the full name for the class avoids conflicts like between `config_imageregistry_operator_openshift_io.rb` and `config_samples_operator_openshift_io.rb`. If we think that `ImagePrunners` will not conflict with anything, then class name can be shorter. Still keeping current `RESOURCE`.

**Comment by pruan-rht** [comment]
https://github.com/openshift/verification-tests/pull/1106

**Pull request closed without merging**